### ISSUE TITLE
VeridiumID

### ISSUE THREAD

**Issue by muelli** [PrivSec-dev/banking-apps-compat-report#410]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

VeridiumID

### Link to app

https://play.google.com/store/apps/details?id=com.veridiumid.authenticator&hl=en_US

### App version

v3.5.1

### Country of the app

United Kingdom

### Build Number

2024022300

### Device list

Pixel 7 Pro

### Profile app tested in

Work profile (Add details in extra notes below, i.e., device manager app name)

### Google Play installed?

Installed

### Where did you install this app from?

Google Play Store

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### Native code debugging

- [X] Allowed
- [ ] Blocked

### Exploit protection compatibility mode

- [X] Enabled
- [ ] Disabled

### Stock OS compatibility

- [X] Works
- [ ] Does not work
- [ ] Not tested

### NFC payments

- [ ] Works
- [ ] Works but requires another service
- [ ] Does not work
- [X] N/A (Not supported by app)
- [ ] Not tested

### Description of the app's functionality

The app does not "add a profile" because "the device is compromised"

**Comment by spring-onion** [comment]
Hi, thanks for the report. Seems like they're based in the UK instead of Germany?

**Comment by muelli** [comment]
yeah, it's actually not a banking app; sorry for having used this venue for reporting the incompatibility. It's an authentication product used by enterprises. So the country doesn't matter that much.

**Comment by spring-onion** [comment]
It's alright, we already include nationwide authentication apps so this one is in the scope.

**Comment by muelli** [comment]
Ah, sorry. I don't have access any more :(

**Comment by spring-onion** [comment]
Fair enough. I'm taking it out of the list until someone else comes along.
